src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/presentation/Header.tsx

```diff
@@ -1,49 +1,50 @@
-/* eslint-disable import/first */
 import * as React from 'react';
 import { getLanguageFromKey } from 'altinn-shared/utils';
 import classNames from 'classnames';
-import { useMediaQuery } from '@material-ui/core';
 import { PresentationType, ProcessTaskType } from '../../types';
+import { Grid } from '@material-ui/core';
 import { ILanguage } from 'altinn-shared/types';
 
 export interface IHeaderProps {
   type: ProcessTaskType | PresentationType;
   language: ILanguage;
   header?: string;
+  appOwner?: string;
```

Comment:
Shouldn't theese be?

```suggestion
  header?: React.ReactNode;
  appOwner?: React.ReactNode;
```

Reply:
Have actually not opened up for markdown in these "reserved" resources, so just now they will never be anything but strings.